bundles/com.ctrlflow.aer.client.log4j2/src/main/java/com/ctrlflow/aer/client/log4j2/AerAppender.java

```diff
@@ -223,8 +223,12 @@ protected Bundle toBundle(ExtendedClassInfo classPackagingData) {
 
     private void sendIncident(Incident incident) {
         try {
-            SimpleAerClient.send(incident, url);
-        } catch (IOException e) {
+			String send = SimpleAerClient.send(incident, url);
+			if (send.contains("Error")) {
```

Comment:
I don’t think this simple heuristic is a good idea. Instead, I propose that we (as in “Codetrails”) update the `SimpleAerClient` to return an object-representation of the server’s JSON response. (This is also useful for informing the client about the status of the reported error.)

That being said, all communication errors should **already** be communicated by a non 2xx HTTP status code, which causes `SimpleAerClient` to throw an exception. If that’s not the case, please report a bug.